### PR TITLE
add parcel data

### PR DESCRIPTION
**Pull request by erikarasnick**
@cole-brokamp check out the readme for this data and let me know if you think we should summarize/aggregate any of the variables differently. 

I also included all 3 of the data packages in the `parcel` repo together here, but can separate them if you think that would be better.

### CODE REVIEW

File: inst/codec_data/parcel/parcel.R [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  tibble::as_tibble() |>
+  select(parcel_id, centroid_lat, centroid_lon, land_use, condo_id, market_total_value, acreage, homestead, rental_registration)
+
+online_parcels <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/auditor_online_parcels-v0.2.0/auditor_online_parcels-v0.2.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble()
+
+property_code_enforcements <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/property_code_enforcements-v1.0.0/property_code_enforcements-v1.0.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble() |>
```
Comment (cole-brokamp) [inline inst/codec_data/parcel/parcel.R:21]: with the newer version of dpkg, converting to tibble before using dplyr shouldn't be needed; I just wanted to make sure it was working as expected...

File: inst/codec_data/parcel/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Parcel Characteristics
+
+Census tract-level measures of parcel characteristics for all residential parcles in Hamilton County, Ohio. Tract-level measures are derived from the data packages stored in the [`parcel` repository](https://github.com/geomarker-io/parcel): `cagis_parcels`, `auditor_online_parcels`, and `property_code_enforcements`. View the metadata for each of these data packages for more information about their sources. 
+
+Parcel-level measures of market total value, acreage, year built, and number of rooms were summarized at the tract level using the median. Because not all parcels in a tract have housing violations, violations were summarized by calculating the number of violations (from 2104 to present) per parcel within each tract. Parcel land use was summarized by grouping more specific land use codes into more general categories (apartments, assisted housing, condominiums, single family homes, two to three family homes, and other) then calculating the fraction of parcels of each type. The fraction of parcels with a homestead flag was also included.
```
Comment (cole-brokamp) [inline inst/codec_data/parcel/README.md:5]: I think this would be clearer as bullet points.

File: inst/codec_data/parcel/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Parcel Characteristics
+
+Census tract-level measures of parcel characteristics for all residential parcles in Hamilton County, Ohio. Tract-level measures are derived from the data packages stored in the [`parcel` repository](https://github.com/geomarker-io/parcel): `cagis_parcels`, `auditor_online_parcels`, and `property_code_enforcements`. View the metadata for each of these data packages for more information about their sources. 
+
```
Comment (cole-brokamp) [inline inst/codec_data/parcel/README.md:4]: Add what versions of each package this uses?

Version 0.1.0 of the `parcel` CoDEC data resource harmonizes `cagis_parcels-v1.1.0`, `auditor_online_parcels-v0.2.0`, and `property_code_enforcements-v1.0.0`.

File: inst/codec_data/parcel/parcel.R [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  st_transform(st_crs(cincy::tract_tigris_2010)) |>
+  st_join(cincy::tract_tigris_2010) |>
+  st_drop_geometry()
+
+d_land_use <- 
+  d |>
+  # collapse land_use categories
+  mutate(land_use_collapsed = 
+    case_when(
+      land_use %in% c("apartment, 40+ units", "apartment, 4-19 units", "office / apartment over", "apartment, 20-39 units") ~ "apartment",
```
Comment (cole-brokamp) [inline inst/codec_data/parcel/parcel.R:43]: add this to readme

- apartment: `apartment, 40+ units`, `apartment, 4-19 units`, `office / apartment over`, ....
- single_family_dwelling: `single family dwelling`

and so on...

**Review by cole-brokamp — CHANGES_REQUESTED** [review]

**Comment by erikarasnick** [comment]
decide if we want to release to s3 or gh releases?

**Pull request merged**